externalLibrary/src/Decoders.elm

```diff
@@ -0,0 +1,172 @@
+module Decoders exposing (..)
```

Comment:
It looks like only `exit` and `item` decoders are used here - making those exports explicit should might make it easier for people looking at the example to understand which parts of this file are most important.